src/robusta/core/sinks/robusta/dal/supabase_dal.py

```diff
@@ -163,6 +165,8 @@ def get_active_services(self) -> List[ServiceInfo]:
                 service_type=service["type"],
                 namespace=service["namespace"],
                 classification=service["classification"],
+                images=service["images"] if service["images"] is not None else [],
```

Comment:
why can it be None? because of older versions? Then you can say instead `service["labels"] if "labels" in service else {}`

Reply:
also in that's the case please add a comment that you check this to support older versions of Robusta

Reply:
yes, it's for backward compatibility
For old stored services the "labels" key exists, it's just contains the value None

That's why I explicitly checked if the value is None